docs/framework/install/net-framework-not-found.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Troubleshooting 'This application could not be started'
+description: Learn what to do if you see a 'This application could not be started' dialog box.
+author: rpetrusha
+ms.author: ronpet
+ms.date: 09/05/2019
+---
+# Troubleshooting a 'This application could not be started' error message
+
+Applications that are developed for the .NET Framework typically require that a specific version of the .NET Framework be installed on your system. In some cases, you may attempt to run an application without either an installed version or the expected version of the .NET Framework present. This often produces an error dialog box like the following:
+
+![This application could not be started](media/app-could-not-be-started.png)
+
+This typically indicates one of the following conditions:
+
+- A .NET Framework installation on your system has become corrupted.
+
+- The version of the .NET Framework needed by your application cannot be detected.
+
+To address this issue so that you can run your application, do the following:
+
+1. Download the [.NET Framework Repair Tool (NetFxRepairTool.exe)](https://www.microsoft.com/en-us/download/details.aspx?id=30135). The tool runs automatically when the download completes.
+
+1. If the .NET Framework Repair Tool recommends any additional action, such as those shown in the following figure, select **Next**.
+
+   ![Recommended changes](media/repair-tool-recommended-changes.png)
+
+1. If the .NET Framework Repair Tool is unable to diagnose and resolve the problem, it displays a a dialog box like the following:
+
+   ![Unable to resolve the problem](media/repair-tool-no-resolution.png)
+
+1. Attempt to launch the application. This should succeed if the .NET Framework Repair Tool has identified and corrected a corrupted .NET Framework installation.
+
+1. If you still cannot run the application, install the latest version of the .NET Framework supported by your version of Windows, as shown in the following table.
+
+   |Windows version|.NET Framework installation|
+   |---|---|
+   |Windows 10 Anniversary Update and later versions|[.NET Framework 4.8 Runtime](https://dotnet.microsoft.com/download/dotnet-framework/net48)|
+   |Windows 10 November Update|[.NET Framework 4.6.2](https://www.microsoft.com/en-us/download/details.aspx?id=53345)|
+   |Windows 10|[.NET Framework 4.6.2](https://www.microsoft.com/en-us/download/details.aspx?id=53345)|
```

Comment:
Not sure about this one, but I see `Windows 10 November Update` and `Windows 10` has the same link. I think there should be either a different link or just keep only `Windows 10`.

Reply:
I've combined them into a single entry, @Youssef1313.